cpp/cmake_modules/BuildUtils.cmake

```diff
@@ -657,6 +657,21 @@ function(ADD_TEST_CASE REL_TEST_NAME)
     set(ARG_LABELS unittest)
   endif()
 
+  foreach(LABEL ${ARG_LABELS})
+    # ensure there is a cmake target which exercises tests with this LABEL
+    set(LABEL_TEST_NAME "test-label-${LABEL}")
```

Comment:
Can we use `test-` instead of `test-label-` as prefix?
`test-label-` is a bit strange for me.

Reply:
Sure